MapboxMobileEvents/MMEEventLogger.h

```diff
@@ -10,6 +10,8 @@ NS_ASSUME_NONNULL_BEGIN
 + (void)setEnabled:(BOOL)enabled;
```

Comment:
We could probably make `enabled` a property.

Reply:
I forgot that this class has so far only used class methods to invoke logging (e.g. `[MMEEventLogger logEvent:debugEvent]`). This PR introduces the notion of using an instance of `MMEEventLogger` in `MMEEventsManager`. 

I think we should change the methods to make them instance based and add a singleton interface so that we can use a `[MMEEventLogger sharedLogger]` instance everywhere we want log messages to go to the same file with the same date. We can still create 1-off instances for other kinds of logging we may eventually want to do in different files.